g.cimis.daily_solar.sh

```diff
@@ -114,8 +114,29 @@ function get_image_interval_list() {
   echo ${list:0:-1}
 }
 
+function verify_or_remove_B2() {
```

Comment:
This function is the main change.  Each B2 image is tested, and if there are nulls, then then `r.stats` count will not match the mask count.  The image is removed from the system.  

Note if you rerun the program, the cloud values will be fetched again, and retested.